src/services/dacs/dacs.hooks.js

```diff
@@ -85,6 +85,28 @@ const isDacAllowed = () => context => {
   return context;
 };
 
+const checkToken = () => context => {
```

Comment:
duplicate of `isProjectAllowed` code. but also probably not needed b/c dacs don't have a token